addon/mixins/flexberry-maplayer-actions-handler.js

```diff
@@ -178,6 +177,60 @@ export default Ember.Mixin.create({
     },
 
     /**
+      Handles {{#crossLink "FlexberryMaplayerComponent/sendingActions.fitBounds:method"}}flexberry-maplayers component's 'fitBounds' action{{/crossLink}}.
+      Fits leaflet map to bounds of selected maplayer.
+
+      @method actions.onMapLayerFitBounds
+      @param {String} boundsPropertyPath Path to a property, which value must be used within action.
+      @param {Object} e Action's event object.
+      @param {Object} e.originalEvent [jQuery event object](http://api.jquery.com/category/events/event-object/)
+      which describes button's 'click' event.
+
+      @example
+      templates/my-form.hbs
+      ```handlebars
+        {{flexberry-maplayers
+          name="Tree node fit bounds button"
+          opacity=layer.options.opacity
+          fitBounds=(action "onMapLayerFitBounds" "layer.options.bounds")
+        }}
+      ```
+
+      controllers/my-form.js
+      ```javascript
+        import Ember from 'ember';
+        import FlexberryMaplayerActionsHandlerMixin from 'ember-flexberry-gis/mixins/flexberry-maplayers-actions-handler';
+
+        export default Ember.Controller.extend(FlexberryMaplayerActionsHandlerMixin, {
+        });
+      ```
+    */
+    onMapLayerFitBounds(...args) {
```

Comment:
```
onMapLayerFitBounds([boundsPropertyPath]) {
```
will be more readable

Reply:
...args allow to put additional params anywhere by developer needs

Reply:
here you use only one param from args, why not do 
```
 let boundsPropertyPath = args[0];
``` 
already in method declaration

Reply:
I think that following the whole structure is more important than slightly less complicated code